.circleci/config.yml

```diff
@@ -1,17 +1,16 @@
-# Golang CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-go/ for more details
-version: 2
+# Check https://circleci.com/docs/language-go/ for more details
+version: 2.1
```

Comment:
@johannesboyne why do you use CircleCI and not github actions?

Reply:
when I started this project, Github Actions did not exist yet ;-)

Reply:
See https://github.com/johannesboyne/gofakes3/pull/105